app/services/proofing/aamva/request/verification_request.rb

```diff
@@ -84,9 +84,36 @@ def add_user_provided_data_to_body
             inside: '//dldv:verifyDriverLicenseDataRequest',
           )
 
+          if IdentityConfig.store.aamva_send_id_type
```

Comment:
This is feature-flagged so we can disable this if for some reason it goes awry.

Reply:
👍 We should add a ticket to remove the feature flag if we don't encounter any issues having it enabled.